ci/glci/alicloud.py

```diff
@@ -54,10 +54,12 @@ def __init__(
         self.acs_client = acs_client
         self.release = release
         self.build_cfg = build_cfg
-        self.image_oss_key = f"gardenlinux-{self.release.version}.qcow2"
-        self.region = build_cfg.alicloud_region
-        self.bucket_name = build_cfg.oss_bucket_name
-        self.image_name = f"gardenlinux-{self.release.canonical_release_manifest_key_suffix()}"
+        if build_cfg is not None:
+            self.bucket_name = build_cfg.oss_bucket_name
```

Comment:
I think it would be better to either require build_cfg not to be None, or to initialise the fields with None in that case - otherweise, code using this object will have to guard against the absense of the field